pages/care.html

```diff
@@ -0,0 +1,144 @@
+          <li>
+            <a class="menu-link" href="#">Care</a>
+          </li>
+          <li>
+            <a class="menu-link" href="facts.html">Facts</a>
+          </li>
+          <li>
+            <a class="menu-link" href="gallery.html">Gallery</a>
+          </li>
+        </ul>
+      </nav>
+  </header>
+  <main>
+    <div class="intro">
+      <div id="summary">
+          <p> This is a quick guide on how to take care of your plants!</p>
+      </div>
+      <div id="table-of-contents">
+        <h2>Plant Care</h2>
+        <ol>
+          <li><a href="#Light">Light 🌞</a></li>
+          <li><a href="#Watering">Watering 💦</a></li>
+          <li><a href="#Soil">Soil and Potting 🪴</a></li>
+          <li><a href="#Temperature">Temperature and Humidity 🌡</a></li>
+        </ol>
+        <h3>Other tips and tricks</h3>
+        <ul>
+          <li><a href="#Troubleshooting">Troubleshooting common issues</a></li>
+          <li><a href="#Green_Thumb">Taking your Green Thumb up a notch!</a></li>
+        </ul>
+      </div>
+    </div>
+    
+    <div class="content">
+
+      <div id="Light">
+        <div class="text">
+          <h3>Light 🌞</h3>
+          <p> Different plants thrive under different amounts of light so it's important to know how much your specific plant needs. Most indoor plants prefer indirect light.</p>
+          <ul>
+            <li><strong>Shade/Low light:</strong> Very little natural light; far from light sources</li>
+            <li><strong>Low Indirect Light:</strong>  Light that's filtered or reflected; a dim room with a north-facing window or a spot far from a window</li>
+            <li><strong>Bright Indirect Light aka "Medium Light":</strong> A bright light source, but not in the direct path of the sun. Most common for house plants!</li>
+            <li><strong>Partial Sun:</strong>  Direct sun, but not intense "peak" sun heat/light</li>
+            <li><strong>Full Sun:</strong> Direct, unfiltered sun for several hours a day.
+            </li>
+          </ul>
+        </div>
+        <img src="/assets/lighting_guide.jpg" alt="lighting guide" width="400" height="350">
+        
+      </div>
+      
+      
+      <div id="Watering">
+        <div class="text">
+          <h3>Watering💦</h3>
+          <p>
+            Identify your specific plant's watering needs. A rule of (green) thumb is that if the top inch of soil is dry, it's time to water!
+            Make sure your plants have proper drainage holes or else standing water can lead to root rot
+          </p>
+        </div>
+        <img src="/assets/watering_chart.jpg" alt="watering guide" height="500" width="350">
+      </div>
+
+      <div id="Soil">
+        <div class="text">
+          <h3>Soil and Potting</h3>
+          <p>
+            <strong>Use the Right Soil:</strong> Most houseplants do well in a general-purpose potting mix. However, some plants like succulents or cacti need a fast-draining soil mix to prevent root rot.
```

Comment:
The Strong Importance element "indicates that its contents have strong importance, seriousness, or urgency. Browsers typically render the contents in bold type".

 It has semantic meaning and serves to draw attention to something. Therefore, it should not be used to apply bold styling. to make the text bold, apply styling with CSS. 

Consider each `strong` used in this project. Was the element used for its intended purpose or should it be removed and `font-weight` in a stylesheet be used instead?

See docs [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong)